__snapshots__/flex-recycle.test.js

```diff
@@ -72,38 +89,38 @@ You must be authenticated.
 `
 
 exports['flex recycle by specifying credentials as options when valid and existent serviceId should succeed 1'] = `
-debug:  Checking for package updates
-debug:  Request:  POST http://localhost:3234/session
-debug:  Response: POST http://localhost:3234/session 200
-debug:  Request:  POST http://localhost:3234/v2/jobs
-debug:  Response: POST http://localhost:3234/v2/jobs 200
-info:  Recycle initiated, received job idOfJobThatIsRecyclingTheService
-debug:  Writing JSON to file projectSetupPath
-debug:  Writing contents to file projectSetupPath
-info:  Saved job ID to project settings.
-debug:  Request:  DELETE http://localhost:3234/session
-debug:  Response: DELETE http://localhost:3234/session 204
-debug:  Logged out current user.
+[debug] Checking for package updates
+[debug] Request:  POST http://localhost:3234/session
+[debug] Response: POST http://localhost:3234/session 200
+[debug] Request:  POST http://localhost:3234/v2/jobs
+[debug] Response: POST http://localhost:3234/v2/jobs 200
+[debug] Writing JSON to file projectSetupPath
+[debug] Writing contents to file projectSetupPath
```

Comment:
No, it isn't new. The first comes from the function, responsible to convert some data to JSON. Converting to JSON might fail... This function then calls another that writes to file.

Reply:
So, do we need both? If not, please remove one of them, even if the code is not new. Otherwise we should either create a JIRA task (an overkill for this, IMO) or we will forget about that.

Reply:
Not entirely sure. It seems a bit too verbose but maybe not redundant as they happen at different steps of code execution.